### PR TITLE
Clarify usage of url-safe base64

### PR DESCRIPTION
**Pull request by junderw**
This gave me some trouble. It would help if people were warned about this.

### CODE REVIEW

File: templates/faraday_rest.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ can be found here:
 {% endfor %}
 
 **NOTE**: The `byte` field type must be set as the base64 encoded string
-representation of a raw byte array.
+representation of a raw byte array. Also, any time this must be used in a URL path
```
Comment (guggero) [inline templates/faraday_rest.md:21]: This is sometimes called "URL safe base64 encoding". Maybe we should mention that so people can find that more easily in the programming language they use?

**Review by guggero — APPROVED** [review]
Thank you, this has been long overdue! LGTM, just one nit.

File: templates/faraday_rest.md (hunk 1 above)
Comment (junderw) [reply templates/faraday_rest.md:21]: https://tools.ietf.org/html/rfc4648#section-5

Comment (junderw) [reply templates/faraday_rest.md:21]: I'll add this link in.

Comment (junderw) [reply templates/faraday_rest.md:21]: ok, done.

**Review by guggero — APPROVED** [review]
LGTM, thank you!

**Pull request merged**